vendor/vendor.json

```diff
@@ -526,6 +550,14 @@
 			"revision": "2b8ed96d2a422bf205adcdfa05272dc12c81613b",
 			"revisionTime": "2017-11-08T19:22:26Z"
 		},
+		{
+			"checksumSHA1": "B9epChtmokc876grlgb/Wip3o/M=",
+			"origin": "github.com/alibaba/libnetwork",
+			"path": "github.com/docker/libnetwork",
+			"revision": "9ba7c0d0a2789981612e569a403ad54f51b5b541",
+			"revisionTime": "2018-05-30T16:26:38Z",
+			"tree": true
```

Comment:
In pouchd, we will set hook `libnetwork-setkey` for runtime, it uses the `reexec` package to do this, if vendor package is different from pouchd, it will make mistake.

Reply:
I've addressed this. But I'm not sure about what to do next, should we check in all packages into our repo except `github.com/docker/docker/pkg/reexec`?

I just give you push access to my fork, feel free to make changes.

Reply:
Just govendor libnetwork without `-tree`, and govendor dependent packages one by one base on the `vendor.conf` in libnetwork.